localization/english/wc_game_concepts_l_english.yml

```diff
@@ -91,4 +91,11 @@
  game_concept_obey_legion:0 "Obey the Legion"
 
  game_concept_scourge_damned:0 "Jailed"
- game_concept_scourge_damned_desc:0 "A [relation|E] indicating the [GetTrait('lich_king').GetName( GetNullCharacter )] overseen by the [scourge_jailors|E].\n\n$game_concept_scourge_jailor_and_damned_common_desc$"
\ No newline at end of file
+ game_concept_scourge_damned_desc:0 "A [relation|E] indicating the [GetTrait('lich_king').GetName( GetNullCharacter )] overseen by the [scourge_jailors|E].\n\n$game_concept_scourge_jailor_and_damned_common_desc$"
+
+ game_concept_plague_scheme:0 "Plague Scheme"
+ game_concept_plague:0 "Plague"
+ game_concept_plagued:0 "Plagued"
+ game_concept_plague_scheme_desc:0 "A [hostile_scheme|E] to provoke a disease outbreak that turns people [GetTrait('being_undead').GetName(GetNullCharacter)]. It may spread to neighboring provinces.\n\nThe [scourge|E] start a [war|E] and may take [plagued|E] [counties|E] if won. Also, every [plagued|E] [county|E] gives [special_troops|E] based on [development|E].\n\nIf the $game_concept_plague_scheme$ is successful, [agents|E] convert to [scheme_owner_possessive|E] [faith|E].\n\nOnly [GetTrait('creature_human').GetName(GetNullCharacter)] characters and $human_group$ provinces are vulnerable."
```

Comment:
[plagued|E] leads to the Plague Scheme, so a link in this text is unnecessary.

Reply:
Fixed in 86637f176b523b950f7f3cd09b2c65c15d8391b7.